Keas.Mvc/ClientApp/components/Equipment/SearchEquipment.tsx

```diff
@@ -107,36 +110,39 @@ export default class SearchEquipment extends React.Component<IProps, IState> {
         else {
             // if teamId is not set, this is a new equipment
             this.props.onSelect({
-                attributes: !!equipmentLabel.equipment ? equipmentLabel.equipment.attributes : 
+                attributes: equipmentLabel.equipment.attributes,
+                id: equipmentLabel.equipment.id,
+                make: equipmentLabel.equipment.make,
+                model: equipmentLabel.equipment.model,
+                name: equipmentLabel.equipment.name,
+                serialNumber: equipmentLabel.equipment.serialNumber,
+                space: equipmentLabel.equipment.space,
+                tags: equipmentLabel.equipment.tags,
+                teamId: equipmentLabel.equipment.teamId,
+                type: equipmentLabel.equipment.type
```

Comment:
only if you want all of the props copied over, but you could alway use the spread operator here to copy all the values over w/e specifying each individual one, just like `{ ...equipmentLabel.equipment }`